aea/connections/oef/connection.py

```diff
@@ -377,58 +353,18 @@ def send(self, envelope: Envelope) -> None:
         :param envelope: the message.
         :return: None
         """
-        if envelope.protocol_id == "default":
-            self.send_default_message(envelope)
-        elif envelope.protocol_id == "fipa":
-            self.send_fipa_message(envelope)
+        if envelope.protocol_id == "gym":
```

Comment:
@marcofavorito maybe the connections should also have an `excluded_protocols` array in the configs?

Reply:
yes that makes sense! Do you mind opening an issue?